.bazelproject

```diff
@@ -4,13 +4,13 @@ directories:
   build_defs
   tools
   third_party
-#  particles
-#  src
+  particles
```

Comment:
Merge this change separately? Presume someone accidentally committed this?

Reply:
Accident. Removing these from .bazelproject massively speeds up IntelliJ.